src/pages/comparison/comparison.tsx

```diff
@@ -175,11 +196,32 @@ const RightLayersPanelWrapper = styled(LeftLayersPanelWrapper)`
   })};
 `;
 
+const getLayerExamples = (): LayerExample[] => Object.values(EXAMPLES);
```

Comment:
It should be already array. I made refactoring on `EXAMPLES`

Reply:
Done